selftest/cgroup/main.bpf.c

```diff
@@ -14,8 +14,8 @@ struct {
 } events SEC(".maps");
 long ringbuffer_flags = 0;
 
-SEC("cgroup/socket")
-int cgroup__socket(struct bpf_sock *sk)
+SEC("cgroup/sock")
```

Comment:
Where can I find a reference to accepted SEC macro handles?

Reply:
There: https://github.com/libbpf/libbpf/blob/a0d1e22c7790f22809d33b3c5c1e3ded89157cc8/src/libbpf.c#L8472-L8554